staging_manifest.yml

```diff
@@ -13,9 +13,10 @@ services:
 - federalist-staging-s3
 - federalist-staging-env
 env:
-  NODE_ENV: staging
+  NODE_ENV: production
+  APP_ENV: staging
   SAILS_LOG_LEVEL: verbose
-  NPM_CONFIG_PRODUCTION: false
+  NPM_CONFIG_PRODUCTION: true
```

Comment:
You may be able to refactor all this to only use `NPM_CONFIG_PRODUCTION` to true and leave your `NODE_ENV: staging` in order to not need those `APP_ENV` references. 💭 

source: https://devcenter.heroku.com/articles/nodejs-support
> When `NPM_CONFIG_PRODUCTION` is true, npm automatically runs all scripts in a subshell where NODE_ENV is 'production.’

Since it uses a version of Heroku buildpack, it can be safe to assume this works. I would test it.